pkg/controller/import-controller.go

```diff
@@ -358,6 +365,43 @@ func (r *ImportReconciler) reconcilePvc(pvc *corev1.PersistentVolumeClaim, log l
 	return reconcile.Result{}, nil
 }
 
+func (r *ImportReconciler) copyImportProxyConfigMap(pvc *corev1.PersistentVolumeClaim, pod *corev1.Pod) error {
+	cdiConfig := &cdiv1.CDIConfig{}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: common.ConfigName}, cdiConfig); err != nil {
+		return err
+	}
+	cmName, err := GetImportProxyConfig(cdiConfig, common.ImportProxyConfigMapName)
+	if err != nil || cmName == "" {
+		return nil
+	}
+	cdiConfigMap := &corev1.ConfigMap{}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: r.cdiNamespace}, cdiConfigMap); err != nil {
+		return err
+	}
+	importConfigMap := &corev1.ConfigMap{}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: pvc.Namespace}, importConfigMap); err != nil {
+		if !k8serrors.IsNotFound(err) {
+			return err
+		}
+		importConfigMap.Name = cmName
```

Comment:
I would make the name unique per importer pod, otherwise you will have name clashes or attempts to delete the cm while multiple imports are running. Also if you make the name unique (use the importer, dv, or pvc name as part of the cm name?) then you don't have to check if it exists, you can just create it (and ignore already existing failures), and you don't need list/watch/update RBAC on config maps.

Reply:
Done, and removed the "update" RBAC. However, for getting the cdi namespace ConfigMap, it requires not only “get” but also both “list “ and “watch”.
```
I1207 23:46:17.113513       1 reflector.go:255] Listing and watching *v1.ConfigMap from pkg/cache/internal/informers_map.go:250
W1207 23:46:17.115447       1 reflector.go:324] pkg/cache/internal/informers_map.go:250: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:cdi:cdi-sa" cannot list resource "configmaps" in API group "" at the cluster scope
E1207 23:46:17.115491       1 reflector.go:138] pkg/cache/internal/informers_map.go:250: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:cdi:cdi-sa" cannot list resource "configmaps" in API group "" at the cluster scope
```
Note that since the config-controller watches CDIConfig, it gets "list" and "watch" for free.